apps/L1E/instagram/instagram-frontend/src/app/(main)/create/page.tsx

```
@@ -1,7 +1,114 @@
+    const file = e.target.files?.[0];
+    if (file) {
+      const reader = new FileReader();
+      reader.onload = (event) => {
+        setSelectedImage(event.target?.result as string);
+        setPostStep('add-caption');
+      };
+      reader.readAsDataURL(file);
+    }
+  };
+
+  const handlePostClick = () => {
```

ESLint

[error] 'handlePostClick' is assigned a value but never used.

(no-unused-vars)